RoboFile.php

```diff
@@ -169,10 +169,10 @@ public function runTests($repoOwner, $repoName, $repoBranch)
 		$dockerTesting = new TestContainer;
 		$dockerTesting->set('name', 'client');
 		$dockerTesting->set('network', $dockerNetwork);
-		$dockerTesting->set('daemon', false);
+//		$dockerTesting->set('daemon', false);
 		$dockerTesting->set(
 			'ports', array(
-				'5900' => '5901'
+				'5901' => '5900'
```

Comment:
I recall doing this change but still having the problem, so it works for you, right?